lib/main_common.pm

```diff
@@ -303,13 +303,14 @@ sub is_leanos {
 
 sub is_desktop_module_selected {
     # desktop applications module is selected if following variables have following values:
-    # productivity and ha require desktop applications, so it's preselected
+    # ha require desktop applications on sle <15, so it's preselected
     # same is true for sles4sap
     return
       get_var('ADDONS', '') =~ /all-packages|desktop|we/
       || get_var('WORKAROUND_MODULES', '') =~ /desktop|we/
       || get_var('ADDONURL',           '') =~ /desktop|we/
-      || get_var('SCC_ADDONS',         '') =~ /desktop|we|productivity|ha/
+      || (!is_sle('15+') && get_var('SCC_ADDONS', '') =~ /desktop|we|productivity|ha/)
```

Comment:
let's use `is_sle('<15') here instead of negation. Other than that LGTM

Reply:
keep in mind that these are not equivalent. In the case here the `is_sle` is also implicit due to the other variables but it will not be true in general.